src/azul/__init__.py

```diff
@@ -309,6 +309,17 @@ def cart_item_state_machine_name(self):
     def cart_api_ip_whitelist(self):  # TODO: Remove when authentication is added
         return os.environ['AZUL_CART_API_IP_WHITELIST'].split(' ')
 
+    @property
+    def access_token_issuer(self):
+        return "https://humancellatlas.auth0.com/"
```

Comment:
```suggestion
        return "https://humancellatlas.auth0.com"
```

If at all possible. We use base URLs that don't end in slash because …